### PR TITLE
Update Empty Application template for beta6.  Closes #275.

### LINKED ISSUE #275: ASP.NET5 Beta6 Empty Web Template
Sample is uploaded http://1drv.ms/1LAz7uJ

### PR DESCRIPTION
**Pull request by Spinarooni**

### CODE REVIEW

File: templates/projects/empty/project.json [hunk 1]
```diff
@@ -3,9 +3,9 @@
   "version": "1.0.0-*",
 
   "dependencies": {
-    "Microsoft.AspNet.Server.IIS": "1.0.0-beta5",
-    "Microsoft.AspNet.Server.WebListener": "1.0.0-beta5",
-    "Kestrel": "1.0.0-beta5"
+    "Microsoft.AspNet.Server.IIS": "1.0.0-beta6",
+    "Microsoft.AspNet.Server.WebListener": "1.0.0-beta6",
+    "Kestrel": "1.0.0-*"
```
Comment (sayedihashimi) [inline templates/projects/empty/project.json]: Can we specify 1.0.0-beta6 here instead of 1.0.0-* ?

Comment (peterblazejewicz) [inline templates/projects/empty/project.json]: It works that way and that's how it is written in sources.

Comment (Spinarooni) [inline templates/projects/empty/project.json]: I suppose so.  I was basing it on the template files in the Zip provided.  Is there an inconsistency in the template files that were provided?  I noticed the Web App Simple (Basic) project.json is similar (not specifying the beta6 tag).

Comment (peterblazejewicz) [inline templates/projects/empty/project.json]: that is confusing: The Web Api template has `"Kestrel": "1.0.0-beta6",`

Comment (sayedihashimi) [inline templates/projects/empty/project.json]: If the .zip file has * then its wrong. We should specify the version explicitly.

Comment (peterblazejewicz) [inline templates/projects/empty/project.json]: I've used explicit version:
https://github.com/OmniSharp/generator-aspnet/pull/287/files#diff-c1c9f3d699ed004eda6c46c17fab21e5R8
Thansk!

**Comment by Spinarooni** [comment]
I've update the PR to use the Kestrel beta6 reference.

**Comment by sayedihashimi** [comment]
Thanks @Spinarooni, I've merged it to the beta6 branch. Once we release it will be merged to master. Closing this now.

**Pull request closed without merging**